clouddriver-kubernetes/src/test/java/com/netflix/spinnaker/clouddriver/kubernetes/caching/view/provider/KubernetesDataProviderIntegrationTest.java

```diff
@@ -116,8 +117,20 @@
   private static final GlobalResourcePropertyRegistry resourcePropertyRegistry =
       new GlobalResourcePropertyRegistry(
           handlers, new KubernetesUnregisteredCustomResourceHandler());
-  private static final AccountCredentialsRepository credentialsRepository =
-      new MapBackedAccountCredentialsRepository();
+  private static final CredentialsRepository<KubernetesNamedAccountCredentials>
+      credentialsRepository =
+          new MapBackedCredentialsRepository<>(
+              KubernetesProvider.PROVIDER_NAME,
+              new CredentialsLifecycleHandler<KubernetesNamedAccountCredentials>() {
```

Comment:
It would probably be useful to define a `NoopCredentialsLifecycleHandler` that can be re-used in tests, as I'd imagine we might start needing an object like this in a few places.

Reply:
Good idea!